packages/common/react-components/src/components/Dialog/Dialog.tsx

```diff
@@ -89,6 +89,7 @@ export const Dialog = ({
           onCloseAutoFocus={(event) => event.preventDefault()}
           {...slots.content}
           className={mx(
+            'flex flex-col',
```

Comment:
`flex-col` by default can cause very subtle bugs except in those very specific cases where the element has an extrinsic size and one of its children should grow to fill it. Only part of that scenario is present here, so there is no cause to have `flex-col`.

Reply:
If the Dialog that Kai is consuming often has an extrinsic size and `flex-col`, kai can reduce classname repetition in slots by defining its own Dialog component that sets those defaults.